app_flutter/analysis_options.yaml

```diff
@@ -0,0 +1,192 @@
+# Specify analysis options.
+#
+# Until there are meta linter rules, each desired lint must be explicitly enabled.
+# See: https://github.com/dart-lang/linter/issues/288
+#
+# For a list of lints, see: http://dart-lang.github.io/linter/lints/
+# See the configuration guide for more
+# https://github.com/dart-lang/sdk/tree/master/pkg/analyzer#configuring-the-analyzer
+#
+# There are other similar analysis options files in the flutter repos,
```

Comment:
I guess you copied this file from somewhere, is this section still relevant?

Reply:
Good catch! I removed this header and left a comment saying to stay in sync with the flutter/flutter lint rules.